db/migrate/20150226095809_add_status_enums_to_orphan.rb

```diff
@@ -0,0 +1,30 @@
+class AddStatusEnumsToOrphan < ActiveRecord::Migration
+  def up
+    add_column :orphans, :status, :integer, default: 0
+    add_column :orphans, :sponsorship_status, :integer, default: 0
+
+    set_status
+    set_sponsorship_status
+  end
+
+  def down
+    remove_column :orphans, :status
+    remove_column :orphans, :sponsorship_status
+  end
+
+  private
+
+  def set_status
+    Orphan.all.each do |orphan|
+      status_code = orphan.orphan_status.code
+      orphan.update_column(:status, ( status_code - 1 ))
+    end
+  end
+
+  def set_sponsorship_status
+    Orphan.all.each do |orphan|
+      status_code = orphan.orphan_sponsorship_status.code
+      orphan.update_column(:sponsorship_status, ( status_code - 1 ))
+    end
+  end
+end
```

Comment:
I don't see a reason to test migrations unless they are very big to test manually. 
We could write a test to `migrate_down`, populate the DB and then `migrate_up`. And after this goes in production we just remove the test. But this is the same as testing this manually.

Since our production db is small we can clone it and test the migration on our production app locally.